app/src/main/java/org/mifos/mobilebanking/presenters/RegistrationPresenter.java

```diff
@@ -68,7 +68,7 @@ public void onCompleted() {
                     @Override
                     public void onError(Throwable e) {
                         getMvpView().hideProgress();
-                        getMvpView().showError(MFErrorParser.errorMessage(e));
+                        getMvpView().showError(context.getString(R.string.fail_register));
```

Comment:
Do not show a custom error msg by using `context.getString(R.string.fail_register)`, as `MFErrorParser.errorMessage(e)` contains important error msg which should be displayed to the user so revert back to it.